kraft/commands/devel/dependency.py

```diff
@@ -0,0 +1,71 @@
+# SPDX-License-Identifier: BSD-3-Clause
+#
+# Authors: Gaulthier Gain <gaulthier.gain@uliege.be>
+#
+# Copyright (c) 2020, Université de Liège., ULiege. All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+#
+# 1. Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+# 2. Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+# 3. Neither the name of the copyright holder nor the names of its
+#    contributors may be used to endorse or promote products derived from
+#    this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+# SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+import os
+import sys
+
+import click
+
+from kraft.commands.list import update
+from kraft.components.library import Library
+from kraft.components.types import RepositoryType
+from kraft.context import kraft_context
+from kraft.errors import KraftError
+from kraft.logger import logger
+from kraft.utils import op, dir
+
+import platform
+import subprocess
+
+TOOLS_PATH = './package/docker/gotools/tools_'
```

Comment:
This may be dependent on where how the package is installed, the extra binaries can be referenced globally, no?

Reply:
You mean, using an absolute path?

Reply:
No, the `TOOLS_PATH` may just need to call a method to determine where the binaries were installed on the system.  I will check this.